framework/src/modules/chain/chain.js

```diff
@@ -181,7 +181,9 @@ module.exports = class Chain {
 			scope.bus.message('bind', scope);
 
 			// Listen to websockets
-			await scope.webSocket.listen();
+			if (scope.config.peers.enabled) {
+				await scope.webSocket.listen();
```

Comment:
That's for the old peer system. We can't remove this logic right now because the tests would fail. I just added this `if` check because of a bug in the config (if the old peers system is turned off in the main config, it used to throw an error here).